deltachat-ios/AppDelegate.swift

```diff
@@ -295,6 +296,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UNUserNotificationCenterD
                 self?.maybeRegisterForRemoteNotifications()
             }
         }
+        notificationManager = NotificationManager()
```

Comment:
`registerForNotifications()` may be called multiple times; that would overwrite existing NotificationManager objects, not sure if that is what we want to do.

Reply:
fixed